### PR TITLE
Add style checks for i18n.conf files

### PR DESCRIPTION
**Pull request by zmstone**
https://emqx.atlassian.net/browse/EMQX-8873

### CODE REVIEW

File: scripts/check-i18n-style.escript [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+check_desc_string(Name, Tr, <<>>) ->
+    io:format(standard_error, "WARNING: ~s(~s): empty string~n", [Name, Tr]);
+check_desc_string(Name, Tr, BinStr) ->
+    Str = unicode:characters_to_list(BinStr, utf8),
+    Die = fun(Reason) ->
+            die("~s(~s): cannot ~s~n", [Name, Tr, Reason])
+          end,
+    case Str of
+        [$\s | _] ->
+            Die("begin with whitespace");
```
Comment (ieQu1) [inline scripts/check-i18n-style.escript:76]:
```suggestion
            Die("begins with whitespace");
```

File: scripts/check-i18n-style.escript [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        [$\n | _] ->
+            Die("begin with line-break");
```
Comment (ieQu1) [inline scripts/check-i18n-style.escript:78]:
```suggestion
            Die("Remove leading line-break");
```

File: scripts/check-i18n-style.escript [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        [$<, $b, $r | _] ->
```
Comment (ieQu1) [inline scripts/check-i18n-style.escript:79]:
```suggestion
        ["<br" | _] ->
```
I think this will work as well

Comment (zmstone) [reply scripts/check-i18n-style.escript:79]: you mean `"<br" ++ _` ?

**Review by ieQu1 — APPROVED** [review]

**Pull request merged**